addons/l10n_pe/models/account_move.py

```diff
@@ -3,6 +3,20 @@
 from odoo.tools.sql import column_exists, create_column
 
 
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _get_l10n_latam_documents_domain(self):
+        self.ensure_one()
+        result = super()._get_l10n_latam_documents_domain()
+        if self.company_id.country_id.code != "PE" or not self.journal_id.l10n_latam_use_documents or self.journal_id.type != "sale":
```

Comment:
Shouldn't `account_fiscal_country_id` be used instead of `country_id` ?

Reply:
@luistorresm We could also do it immediately in 16.0 to avoid that confusion or use country_code immediately on self.